opm/autodiff/MissingFeatures.cpp

```diff
@@ -55,6 +89,14 @@ namespace MissingFeatures {
             "VAPPARS", "VISCREF", "WATVISCT",
             "WPAVE", "WPIMULT", "WPITAB", "WTEMP",
             "WTEST", "WTRACER", "ZIPPY2" };
+        std::multimap<std::string, PartiallySupported<std::string> > string_options;
+        std::multimap<std::string, PartiallySupported<int> > int_options;
+        addUnsupported<ParserKeywords::COMPORD, ParserKeywords::COMPORD::ORDER_TYPE, std::string>(string_options , "TRACK");
```

Comment:
I realize that now, and see that I should have read the code a bit more carefully. I think that must change: addUnsupported could really be called addSupported! However, instead of maintaining a list of supported features it is better I think to have a list of unsupported parts, so that addUnsupported() works as I initially assumed. Then the checking loop on line 56 should use `==` instead of `!=` and of course the message must be rewritten, and the calls change to supply the complement of the current options (unsupported parts instead of supported parts).

If you do not agree with that (perhaps there are many more unsupported than supported options for this keyword?) then we must rename addUnsupported->addSupported.